.gitignore

```diff
@@ -40,3 +40,6 @@ CMakeUserPresets.json
 
 # Rust
 rust/jank
+/compiler+runtime/llvm-build/
+/compiler+runtime/llvm-install/
+/compiler+runtime/llvm/
```

Comment:
These dirs should be in `build`. I suspect that you didn't `cd build` before running `./bin/build-clang`, which is what lead to this. However, the entire `build` dir is currently ignored, so there should be no change needed here.